program/src/instructions/collect.rs

```diff
@@ -22,10 +22,13 @@ pub struct Collect<'info> {
     // n fee accounts in remaining accounts
     // where the accounts are derived from the passed in
     // indices and bumps, in order
+
+    // TODO: I guess we're okay with having no signers
```

Comment:
Yeah, one option is to make it permissioned to a signer, the other way is to hard-code the destination address. The latter way is nicer because it lets us run a permissionless service to collect fees.